index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
- [ ] There is no style file linked to your HTML. Kindly fix that as i am unable to verify the preview